receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/filesystem_scraper_test.go

```diff
@@ -134,22 +187,48 @@ func TestScrapeMetrics(t *testing.T) {
 
 			assert.GreaterOrEqual(t, metrics.Len(), 1)
 
-			assertFileSystemUsageMetricValid(t, metrics.At(0), fileSystemUsageDescriptor, test.expectedDeviceDataPoints*fileSystemStatesLen)
+			assertFileSystemUsageMetricValid(
+				t, metrics.At(0), fileSystemUsageDescriptor,
+				test.expectedDeviceDataPoints*fileSystemStatesLen, test.expectedDeviceLabelValues,
+			)
```

Comment:
Nit: I think it would be more readable to just put each variable on a separate line at this point (same for below, and for the function signatures)